src/cardano-wallet-adapter/adapter.ts

```diff
@@ -34,20 +34,18 @@ export class CrossmintCardanoWalletAdapter {
 
             const client = CrossmintEmbed.init(this._config);
 
-            const accounts = await client.login();
+            const loginData = await client.login();
 
-            if (accounts === null) {
-                throw new WalletWindowClosedError("User rejected the request");
-            }
-            if (accounts === undefined || accounts.length === 0) {
+            if (loginData?.accounts?.[0] == null) {
                 throw new WalletWindowClosedError("User rejected the request or closed the window");
             }
 
-            const account = accounts[0];
+            //TODO: walletId and deviceId should be implemented when AA supports Cardano
+            const accounts = loginData.accounts[0]
```

Comment:
nit: it's just one `account`, right?
```suggestion
            const account = loginData.accounts[0]
```